cds-snc/covid-alert-portal
# Column sort

To test:

1. Go to the notification history and try sorting with the new column sorting buttons

2. Go to the admin panel and try enabling/disabling the 'can send outbreak alerts' flag for whatever user and test to see if they can see/not see the public exposures tab
